doc/usage.rst

```diff
@@ -29,6 +29,7 @@ NOTE: the output for the following formats are generated in accordance with sche
 * ``json`` follows the `own JSON schema </doc/schemas/fix/schema.json>`_
 * ``junit`` follows the `JUnit XML schema from Jenkins </doc/schemas/fix/junit-10.xsd>`_
 * ``xml`` follows the `own XML schema </doc/schemas/fix/xml.xsd>`_
+* `gitlab` follows the `codeclimate JSON schema </doc/schemas/fix/codeclimate.json>`_
```

Comment:
Can `gitlab` be surrounded with double backticks as the other entries and placed a few lines higher, so the list keeps its alphabetical order?